code/__HELPERS/global_lists.dm

```diff
@@ -239,8 +240,14 @@ GLOBAL_LIST_EMPTY(ignore_health_alerts_from)
 	//Posters
 	paths = subtypesof(/datum/poster) - /datum/poster/wanted
 	for(var/T in paths)
-		var/datum/poster/P = new T
-		GLOB.poster_designs += P
+		var/datum/poster/poster = new T
+		GLOB.poster_designs += poster
+
+	// Aster posters
+	paths = subtypesof(/datum/poster/asters) - /datum/poster/wanted
```

Comment:
`/datum/poster/asters` does not exist.

Reply:
It shouldn't need to exist in of itself, only its subtypes.

Reply:
Nevermind, it appears it does. Apparently compiling isn't the end all according to to CI Suite.

Reply:
You're using VSC with `BYOND DM Language Support` and `DreamMaker Language Client` extensions, right? Right?
If not, then you probably should.